backend/model/User.ts

```diff
@@ -0,0 +1,27 @@
+import { DataTypes, Model } from 'sequelize'
+const sequelize = require('../config/database')
+
+class User extends Model {}
+
+User.init(
+    {
```

Comment:
I would also add a primary key to the User table

Reply:
sequelize adds a default id as a primary key.